### ISSUE TITLE
Suggestion: Windows/Browser Desktop App

### ISSUE THREAD

**Issue by rawpie2** [wukko/cobalt#168]
I enjoy the use of cobalt but integrating it in windows/firefox/chrome etc would be nice so I can just right click or be able to instantly download the video/audio in fewer steps.
Thanks

**Comment by skorpionsk** [comment]
yes! this is so good, it could be like a little menu to paste in the link and select the preferred mode. very good idea!

**Comment by ethrythedev** [comment]
i made a [userscript](https://greasyfork.org/en/scripts/483370-simple-youtube-downloader) for youtube using cobalt before, but i might make an extension using the api some day

i'll leave it here if i get around to it

**Comment by ethrythedev** [comment]
a bookmarklet could also be a good idea, wukko

**Comment by princessmortix** [comment]
I think I also could do a GUI app for cobalt :)

**Comment by herrobriam** [comment]
bump feature
